resources/composer.json

```diff
@@ -32,9 +32,15 @@
         "https://github.com/rchouinard/rych-random/pull/5": "https://patch-diff.githubusercontent.com/raw/rchouinard/rych-random/pull/5.patch",
         "https://github.com/rchouinard/rych-random/pull/7": "https://patch-diff.githubusercontent.com/raw/rchouinard/rych-random/pull/7.patch"
       },
-      "jasig/phpcas": {
-        "phpCAS-1.3.8_handle_XML_parsing_ECAS_attributes.patch": "../resources/patches/phpCAS-1.3.8_handle_XML_parsing_ECAS_attributes.patch"
+      "apereo/phpcas": {
+        "phpCAS-1.4.0_handle_XML_parsing_ECAS_attributes.patch": "../resources/patches/phpCAS-1.4.0_handle_XML_parsing_ECAS_attributes.patch"
       }
     }
+  },
+  "config": {
+    "allow-plugins": {
+      "composer/installers": true,
+      "cweagans/composer-patches": true
+    }
```

Comment:
Hi @julien- ,
Was the composer lock generated using composer version 2 ?
I believe the Drupal 7 websites still needs to use composer in version 1.
Could you please check it?

`As of Composer 2.2.0, the allow-plugins option adds a layer of security...`
https://getcomposer.org/doc/06-config.md#allow-plugins

cc @iulianweb